graphdatascience/model/model_proc_runner.py

```diff
@@ -45,6 +46,34 @@ def create(
             relationship_type_embeddings,
         )
 
+    @compatible_with("train", min_inclusive=ServerVersion(2, 5, 0))
+    @client_only_endpoint("gds.model.transe")
+    def train(self,
+              G: Graph,
+              proportions: list,
+              embedding_dimension: int,
+              batch_size: int,
+              epochs: int,
+              optimizer: str,
+              optimizer_kwargs: dict,
+              # loss: str
+              ) -> int:
+        config = {'scoring_function': 'TransE',
+                  'proportions': proportions,
```

Comment:
I call this `split_ratios` in the python runtime. I think "ratio" is a more common term to refer to this (also what pyKEEN use for example)